### PR TITLE
Allows "value" as an alias for "name"

### PR DESCRIPTION
**Pull request by dorey**
* allows "value" as an alias for "name" (as in old behavior)
* moves existing tests away from xls fixtures

### CODE REVIEW

**Comment by dorey** [comment]
@ukanga @yanokwa @pld @mberg 

This PR restores `value` as an accepted alias for the `name` column of the survey sheet as discussed in [odk-developers](https://groups.google.com/forum/#!topic/opendatakit-developers/BZKVfQkotjo) thread.

File: pyxform/tests_v1/test_sheet_columns.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                    ],
+                model__contains=[
+                    '<bind nodeset="/aliases/q1" type="string"/>',
+                    ],
+                xml__contains=[
+                    '<input ref="/aliases/q1">',
+                    '<label>Question 1</label>',
+                    '</input>',
+                    ])
+''' # uncomment when re-implemented
```
Comment (ukanga) [inline pyxform/tests_v1/test_sheet_columns.py]: Any specific reason why we should keep this commented code in?

**Comment by ukanga** [comment]
:+1:

File: pyxform/tests_v1/test_sheet_columns.py (hunk 1 above)
Comment (dorey) [inline pyxform/tests_v1/test_sheet_columns.py]: I was thinking of fixing it in this PR but i didn't get to it. I kept it in as a note about some incomplete behavior associated with aliases. I could attach it to a new issue to fix when-time.

Comment (ukanga) [inline pyxform/tests_v1/test_sheet_columns.py]: Attaching to a new issue would be the ideal way I think, otherwise this is good to merge.

**Pull request merged**